src/core/src/model.cpp

```diff
@@ -955,7 +955,7 @@ bool ov::Model::has_rt_info(const std::vector<std::string>& args) const {
             return false;
         if (i == args.size() - 1)
             break;
-        const ov::Any& rt_attr = get_rt_arg<std::string>(info, args[i]);
+        const ov::Any rt_attr = get_rt_arg<std::string>(info, args[i]);
```

Comment:
Is there a reason for a copy?

Reply:
yes,
`get_map_from_attr` just performs some checks, casting and returns *a reference*, so lets ignore it in the explanation.
`info = rt_attr` at line 959 performs two actions: destroys data inside `info` and copies data from `rt_attr` into `info`. I don't see if an order of actions is strictly defined, but it looks like on our Windows and MacOS configurations it is Destroy->Copy. If `rt_attr` is a reference to something inside `info`, then at the moment of copy data under `rt_attr` would be already destroyed.

Reply:
Why do you need this change? Do you have a segfault or that? I don't expect such changes...
Moreover below in the line 969 we have the same code and it works well.
Please explain this change.

Reply:
Yes, I had a segfault on macos job and failed test on windows.
During the debug process I found that `info` had empty strings for keys after update.

Reply:
Please create a ticket to understand the root cause